### ISSUE TITLE
Problem with @babel/polyfill

### ISSUE THREAD

**Issue by minhducsun2002** [yeoman/generator-chrome-extension#212]
This happens when I try to run `yo chrome-extension`:
```
...
? Would you like to use UI Action? Page Action
? Would you like more UI Features? Options UI, Content Scripts
? Would you like to set permissions? History, Notifications, Tabs, Web Request
events.js:174
      throw er; // Unhandled 'error' event
      ^

AssertionError [ERR_ASSERTION]: Trying to copy from a source that does not exist: <censored>/node_modules/generator-chrome-extension/node_modules/@babel/polyfill/dist/polyfill.min.js
    at EditionInterface.exports._copySingle (<censored>/node_modules/mem-fs-editor/lib/actions/copy.js:45:3)
    at EditionInterface.exports.copy (<censored>/node_modules/mem-fs-editor/lib/actions/copy.js:23:17)
    at constructor.babel (<censored>/node_modules/generator-chrome-extension/app/index.js:294:13)
    at Object.<anonymous> (<censored>/node_modules/yeoman-generator/lib/base.js:439:23)
    at <censored>/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at <censored>/node_modules/run-async/index.js:24:19
    at <censored>/node_modules/yeoman-generator/lib/base.js:440:9
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
Emitted 'error' event at:
    at Immediate.<anonymous> (<censored>/node_modules/yeoman-generator/lib/base.js:446:16)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

Also, `npm i` complains about `@babel/polyfill` to be deprecated. Could this be the issue?

**Comment by UlisesGascon** [comment]
I assume that this issue is going to be fixed at next release #217, so I close it

**Comment by joelstransky** [comment]
just got this. is there a fix or work around?
